stylesheets/styles.css

```diff
@@ -0,0 +1,363 @@
+    transition: left .4s;
+   }
+  
+  
+  nav ul li:last-child input:focus {
+    border: 2px inset #999;
+    border-bottom: 1px  inset #aaa;
+    border-left: 1px  inset #aaa;
+    border-right: 1px  inset #aaa;
+    background-image: none;
+    outline: none;
+    width: 17rem;
+    color: #555;
+    background-color: white;
+   }
+  
+   nav ul li:last-child input:focus + span {
+    background-image: url(../assets/images/magnify-dark.svg);
+    left: 11%;
+   }
+  
+  nav ul li:not(:last-child):hover{
+    background-image: linear-gradient(to top, rgb(122,122,122) 1% ,rgb(62,62,62)
+    48%, rgb(77,77,77)52%);
+    cursor: pointer;
+  }
+  
+  nav ul li:first-child:hover{
+    background-image: url(../assets/images/nav.svg), linear-gradient(to top, rgb(122,122,122) 1% ,rgb(62,62,62)
+    48%, rgb(77,77,77)52%);
+  }
+  
+  nav ul a {
+    color: #eee;
+    text-decoration: none;
+    font-size: 1.2rem;
+    text-shadow: 1px 1px 1px #444, -1px -1px 1px #444
+  }
+
+  nav ul li a:hover{
+    text-decoration: none;
+  }
+  
+  nav ul li:last-child a img{ 
+    width: 50px;
+  }
+   
+  header {
+    height: 53.8rem;
+    background: url(../assets/images/hero-img.jpg);
+    background-repeat: no-repeat;
+    background-position: center -35rem;
+    background-size: cover; 
+    padding-top: 17rem;
+  }
+  
+  header:hover{
+    cursor: pointer;
+  }
+  
+  header h1{
+    background: url(../assets/images/h1.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 7rem;
+    background-size: auto 10rem;
+    margin-bottom: 1rem;
+  }
+  
+  header h2{
+    background: url(../assets/images/h2.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 7rem;
+    margin-bottom: 1.2rem;
+  }
+  
+  header h3{
+    background: url(../assets/images/cta-simple.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 7rem;
+  }
+  
+  header h3:hover{
+    background: url(../assets/images/cta-hover.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 7rem;
+    cursor: pointer;
+  }
+  
+  aside {
+    display: flex;
+    height: 20.6rem;
+    border-top: 2px solid white;
+  }
+
+  aside > div {
+    flex: 1;
+  }
+  
+  aside div:hover {
+    cursor: pointer;
+  }
+  
+  .product{
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover; 
+    height: 100%;
+    width: 100%;
+    display: block;
+  }
+  
+  .product-gallery div:not(:last-child){
+    border-right: 2px solid white;
+  }
+  
+  .a-1{
+    background-image: url(../assets/images/product-1.jpg);
+  }
+  
+  
+  .a-2 {
+    background-image: url(../assets/images/product-2.jpg);
+  }
+  
+  .a-3 { 
+    background-image: url(../assets/images/product-3.jpg);
+  }
+  
+  .a-4 {
+    background-image: url(../assets/images/product-4.jpg);
+  }
+  
+  footer {
+    /* display: flex; */
+    /* padding: 2.5rem 10rem; */
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.